code/modules/research/anomaly/anomaly.dm

```diff
@@ -88,9 +96,11 @@
 			armour_path = /obj/item/clothing/suit/armor/reactive/teleport
 		if(istype(I, /obj/item/assembly/signaler/anomaly/pyro))
 			armour_path = /obj/item/clothing/suit/armor/reactive/fire
+		if(istype(I, /obj/item/assembly/signaler/anomaly/cryo))
+			armour_path = /obj/item/clothing/suit/armor/reactive/cryo
```

Comment:
It looks like the `anomaly_armour_types` list was never implemented properly. Could you change around this proc a bit so we don't need these ifs? I've written out what I think it should be (needs testing):
```dm
/obj/item/reactive_armour_shell/attackby(obj/item/I, mob/user, params)
	var/static/list/anomaly_armour_types = list(
		/obj/item/assembly/signaler/anomaly/grav = /obj/item/clothing/suit/armor/reactive/repulse,
		/obj/item/assembly/signaler/anomaly/flux = /obj/item/clothing/suit/armor/reactive/tesla,
		/obj/item/assembly/signaler/anomaly/bluespace = /obj/item/clothing/suit/armor/reactive/teleport,
		/obj/item/assembly/signaler/anomaly/pyro = /obj/item/clothing/suit/armor/reactive/fire,
		/obj/item/assembly/signaler/anomaly/cryo = /obj/item/clothing/suit/armor/reactive/cryo,
		/obj/item/assembly/signaler/anomaly/vortex = /obj/item/clothing/suit/armor/reactive/stealth
		)

	if(istype(I, /obj/item/assembly/signaler/anomaly))
		var/obj/item/assembly/signaler/anomaly/A = I
		var/armour_path = anomaly_armour_types[A.type]
		if(!armour_path)
			armour_path = /obj/item/clothing/suit/armor/reactive/stealth //Fallback
		to_chat(user, "<span class='notice'>You insert [A] into the chest plate, and the armor gently hums to life.</span>")
		new armour_path(get_turf(src))
		qdel(src)
		qdel(A)
	return ..()
```
(Notice the added `/obj/item/assembly/signaler/anomaly/vortex = /obj/item/clothing/suit/armor/reactive/stealth` entry in the list.)

Reply:
Tested this and it works, implemented in commit `e1c6354` (steel review commit 2)